### DISCUSSION TITLE
How to FrankenPHP Nextcloud?

### DISCUSSION THREAD

**Discussion by therealresonix** [php/frankenphp#801] [Q&A]
Hey;
How do you setup FrankenPHP and Nextcloud?
Is this even possbile due to massive project Size and complexity?

I want it to use a systemd service and run it local without docker.
I want to keep my own nextcloud user as in PHP-FPM but use a different for the webserver.

**Comment by kesselb** [comment]
We have one more or less successful report here: https://help.nextcloud.com/t/config-file-has-leading-content/194026?u=kesselb

**Reply by therealresonix** [reply]
What does it mean?
